src/kontrol/prove.py

```diff
@@ -349,7 +297,7 @@ def _load_constructor_proof(foundry: Foundry, contract: Contract) -> APRProof:
     return apr_proof
 
 
-def _method_to_initialized_cfg(
+def method_to_initialized_cfg(
```

Comment:
Is this change intentional?

Reply:
I reverted it. This got introduced in a manual merge.